plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -78,11 +79,35 @@ var FbListInlineEdit = new Class({
 			this.setFocus(this.editCell);
 		}.bind(this));
 		
-		// Click outside list clears down selection
+		// Activate edit mode in clicked element
 		window.addEvent('click', function (e) {
+			
+			// First save current current edit if there is one
+			if (this.activeElement !== null) {	
+				// check to see if clicked element is already in edit mode
+				var activeClassName	= this.activeElement.target.className;
+				var activeClasses = activeClassName.split(" ");
+				if (activeClasses[0] == e.target.id){
+					return;
+				}
```

Comment:
Moved this check inside the if - else it would cause error when clicking outside the table as this.activeElement was undefined.